### PR TITLE
Issue #9 Allow code coverage with build tags

### PR DESCRIPTION
**Pull request by award28**
Pull Request for Issue #9

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -83,8 +84,25 @@ func main() {
 		ImageExtension: ".png",
 	}
 	var coverageFloat float64
+
+	if *tagsFlag != "" {
```
Comment (jpoles1) [inline main.go]: What is the purpose of this if statement? Looks like it should be removed, unless I'm mistaken.

File: main.go [hunk 2]
```diff
@@ -83,8 +84,25 @@ func main() {
...
+
+	}
+
+	coverageCommand := ""
+	if *coverageCommandFlag != "" {
+		coverageCommand = *coverageCommandFlag
+		if *tagsFlag != "" {
+			log.Println("Warning: When the covercmd is used the tags flag will be ignored.")
```
Comment (jpoles1) [inline main.go]: Please change to: "Warning: When the covercmd flag is used the tags flag will be ignored."

**Review by jpoles1 — CHANGES_REQUESTED** [review]
Looks great @award28, thanks for your hard work! I've made a few comments, please let me know if you have any questions.

File: main.go (hunk 1 above)
Comment (award28) [reply main.go]: Nope, definitely meant to erase that.

**Comment by award28** [comment]
@jpoles1 I've made the appropriate changes per your requests. Thank you!

**Review by jpoles1 — APPROVED** [review]

**Pull request merged**